app/controllers/employments/AddEmploymentController.scala

```diff
@@ -74,10 +75,11 @@ class AddEmploymentController @Inject()(auditService: AuditService,
     controllers.employments.routes.AddEmploymentController.submitTelephoneNumber().url,
     controllers.employments.routes.AddEmploymentController.cancel().url
   )
-
+  val telephoneRegex = """^(\(?\+?[0-9]*\)?)?[0-9_\- \(\)]*$""".r
   def telephoneNumberSizeConstraint(implicit messages: Messages): Constraint[String] =
     Constraint[String]((textContent: String) => textContent match {
```

Comment:
Can you just use TelephoneNumberConstraint.telephoneNumberSizeConstraint to reduce code duplication?